pylintrc

```diff
@@ -353,6 +353,10 @@ method-rgx=[a-z_][a-z0-9_]{2,}$
 # Naming hint for method names
 method-name-hint=[a-z_][a-z0-9_]{2,}$
 
+# Regular expression which can overwrite the naming style set by typevar-naming-style.
+# If left empty, type variables will be checked with the set naming style.
+typevar-rgx=
```

Comment:
I checked. If it is empty the setting will be `None` (I think this is due to how the config file is parsed). The following lines make sure that the pattern is only picked up if the pattern is set. (Note that this is also how this works for all other patterns).
```python
custom_regex = getattr(self.config, custom_regex_setting_name, None)
if custom_regex is not None:
    regexps[name_type] = custom_regex
```

Reply:
An empty setting will be `re.compile('')` and thus overwrite `regexps[name_type]`. That isn't bad, we just can't leave it empty in `pylintrc`. Otherwise TypeVar names in pylint itself won't be checked. That's similar to the other regex settings in pylintrc, eg. `class-rgx`.

Alternatively, an possibly my preference would be the comment out that line. Similar to `#class-const-rgx=`.

Reply:
Ah I understand why I was mistaken. The functional test don't actually use this `pylintrc` so they passed even with this uncommented.

Commented it out!